tools/serverpod_cli/lib/src/test_util/builders/migration_version_builder.dart

```diff
@@ -27,6 +40,7 @@ class MigrationVersionBuilder {
     return this;
   }
 
+  /// Should include your self in the installedModules.
```

Comment:
Suggestion: Remove comment.

This seems like an instruction for when you were writing the code. If it should stay could you rephrase it so that it is understandable for future devs.

Reply:
Maybe:
```suggestion
  /// The [DatabaseDefinition] should always include its own module in its installedModules.
```

Reply:
But that seems like you should only include other modules?! The point here is you have to add your own project in the modules as well.

Reply:
Yeah, might not be the right phrasing either. But something that makes it a bit more clear what should be included.

/// The [DatabaseDefinition] should always include a reference to itself as a module in its installedModules.